### ISSUE TITLE
Restarting CRC using the daemon does not work on macOS Big Sur

### ISSUE THREAD

**Issue by anjannath** [code-ready/crc#2059]
### General information

  * OS: macOS Big Sur
  * Hypervisor:  hyperkit
  * Did you run `crc setup` before starting it (Yes/No)? Yes

## CRC version
```bash
master
```
 
When starting a stopped CRC instance from the tray (via the daemon) it fails with the following error:

```
time="2021-03-02T20:56:10+05:30" level=debug msg="Received Request:{\"command\":\"start\"}"
time="2021-03-02T20:56:10+05:30" level=info msg="Checking if running as non-root"
time="2021-03-02T20:56:10+05:30" level=info msg="Checking if podman remote executable is cached"
time="2021-03-02T20:56:10+05:30" level=debug msg="Currently podman remote is not supported"
time="2021-03-02T20:56:10+05:30" level=info msg="Checking if admin-helper executable is cached"
time="2021-03-02T20:56:10+05:30" level=debug msg="Running '/Users/redhat/.crc/bin/admin-helper-darwin --version'"
time="2021-03-02T20:56:10+05:30" level=debug msg="Found admin-helper-darwin version 0.0.2"
time="2021-03-02T20:56:10+05:30" level=debug msg="admin-helper executable already cached"
time="2021-03-02T20:56:10+05:30" level=info msg="Checking minimum RAM requirements"
time="2021-03-02T20:56:10+05:30" level=debug msg="Total memory of system is 68719476736 bytes"
time="2021-03-02T20:56:10+05:30" level=info msg="Checking if HyperKit is installed"
time="2021-03-02T20:56:10+05:30" level=debug msg="Running '/Users/redhat/.crc/bin/hyperkit -v'"
time="2021-03-02T20:56:10+05:30" level=debug msg="Found hyperkit version v0.20200224-44-gb54460"
time="2021-03-02T20:56:10+05:30" level=info msg="Checking if crc-driver-hyperkit is installed"
time="2021-03-02T20:56:10+05:30" level=debug msg="Checking if crc-driver-hyperkit is installed"
time="2021-03-02T20:56:10+05:30" level=debug msg="Running '/Users/redhat/.crc/bin/crc-driver-hyperkit version'"
time="2021-03-02T20:56:10+05:30" level=debug msg="Found crc-driver-hyperkit version 0.12.11"
time="2021-03-02T20:56:10+05:30" level=info msg="Checking file permissions for /etc/hosts"
time="2021-03-02T20:56:10+05:30" level=debug msg="Checking if /etc/hosts ownership/permissions need to be adjusted after crc upgrade"
time="2021-03-02T20:56:10+05:30" level=info msg="Checking file permissions for /etc/resolver/testing"
time="2021-03-02T20:56:10+05:30" level=debug msg="Checking file: /Users/redhat/.crc/machines/crc/.crc-exist"
time="2021-03-02T20:56:10+05:30" level=debug msg="Found binary path at /Users/redhat/.crc/bin/crc-driver-hyperkit"
time="2021-03-02T20:56:10+05:30" level=debug msg="Launching plugin server for driver hyperkit"
time="2021-03-02T20:56:10+05:30" level=debug msg="Plugin server listening at address 127.0.0.1:53629"
time="2021-03-02T20:56:10+05:30" level=debug msg="() Calling .GetVersion"
time="2021-03-02T20:56:10+05:30" level=debug msg="Using API Version 1"
time="2021-03-02T20:56:10+05:30" level=debug msg="() Calling .SetConfigRaw"
time="2021-03-02T20:56:10+05:30" level=debug msg="() Calling .GetMachineName"
time="2021-03-02T20:56:10+05:30" level=debug msg="(crc) Calling .GetBundleName"
time="2021-03-02T20:56:10+05:30" level=debug msg="(crc) Calling .GetState"
time="2021-03-02T20:56:10+05:30" level=debug msg="(crc) DBG | time=\"2021-03-02T20:56:10+05:30\" level=debug msg=\"exe=/Users/redhat/.crc/bin/crc-driver-hyperkit uid=0\""
time="2021-03-02T20:56:10+05:30" level=debug msg="(crc) DBG | time=\"2021-03-02T20:56:10+05:30\" level=debug msg=\"hyperkit pid from json: 20624\""
time="2021-03-02T20:56:10+05:30" level=debug msg="(crc) DBG | time=\"2021-03-02T20:56:10+05:30\" level=debug msg=\"hyperkit pid 20624 missing from process table\""
time="2021-03-02T20:56:10+05:30" level=info msg="Starting CodeReady Containers VM for OpenShift 4.7.0..."
time="2021-03-02T20:56:10+05:30" level=debug msg="Updating CRC VM configuration"
time="2021-03-02T20:56:10+05:30" level=debug msg="(crc) Calling .GetConfigRaw"
time="2021-03-02T20:56:10+05:30" level=debug msg="(crc) Calling .GetConfigRaw"
time="2021-03-02T20:56:10+05:30" level=debug msg="(crc) Calling .Start"
time="2021-03-02T20:56:10+05:30" level=debug msg="(crc) DBG | time=\"2021-03-02T20:56:10+05:30\" level=debug msg=\"exe=/Users/redhat/.crc/bin/crc-driver-hyperkit uid=0\""
time="2021-03-02T20:56:10+05:30" level=debug msg="(crc) DBG | time=\"2021-03-02T20:56:10+05:30\" level=debug msg=\"clean start, hyperkit pid file doesn't exist: /Users/redhat/.crc/machines/crc/hyperkit.pid\""
time="2021-03-02T20:56:10+05:30" level=debug msg="(crc) DBG | time=\"2021-03-02T20:56:10+05:30\" level=debug msg=\"Using hyperkit binary from /Users/redhat/.crc/bin/hyperkit\""
time="2021-03-02T20:56:10+05:30" level=debug msg="(crc) DBG | time=\"2021-03-02T20:56:10+05:30\" level=debug msg=\"Using UUID c3d68012-0208-11ea-9fd7-f2189899ab08\""
time="2021-03-02T20:56:11+05:30" level=debug msg="(crc) DBG | time=\"2021-03-02T20:56:11+05:30\" level=debug msg=\"Generated MAC e:f0:e3:c7:4f:be\""
time="2021-03-02T20:56:11+05:30" level=debug msg="(crc) DBG | time=\"2021-03-02T20:56:11+05:30\" level=debug msg=\"Starting with cmdline: BOOT_IMAGE=(hd0,gpt3)/ostree/rhcos-8db996458745a61fa6759e8612cc44d429af0417584807411e38b991b9bcedb9/vmlinuz-4.18.0-240.10.1.el8_3.x86_64 random.trust_cpu=on console=tty0 console=ttyS0,115200n8 ignition.platform.id=qemu ostree=/ostree/boot.0/rhcos/8db996458745a61fa6759e8612cc44d429af0417584807411e38b991b9bcedb9/0 root=UUID=a9fd5b50-93e0-4df8-975f-6e0d412d1183 rw rootflags=prjquota\""
time="2021-03-02T20:56:11+05:30" level=debug msg="(crc) DBG | time=\"2021-03-02T20:56:11+05:30\" level=debug msg=\"Trying to execute /Users/redhat/.crc/bin/hyperkit -A -u -F /Users/redhat/.crc/machines/crc/hyperkit.pid -c 4 -m 9216M -s 0:0,hostbridge -s 31,lpc -s 1:0,virtio-net -U c3d68012-0208-11ea-9fd7-f2189899ab08 -s 2:0,virtio-blk,file:///Users/redhat/.crc/machines/crc/crc.qcow2,format=qcow -s 3,virtio-sock,guest_cid=3,path=/Users/redhat/.crc/machines/crc -s 4,virtio-rnd -l com1,autopty=/Users/redhat/.crc/machines/crc/tty,log=/Users/redhat/.crc/machines/crc/console-ring -f kexec,/Users/redhat/.crc/cache/crc_hyperkit_4.7.0/vmlinuz-4.18.0-240.10.1.el8_3.x86_64,/Users/redhat/.crc/cache/crc_hyperkit_4.7.0/initramfs-4.18.0-240.10.1.el8_3.x86_64.img,earlyprintk=serial BOOT_IMAGE=(hd0,gpt3)/ostree/rhcos-8db996458745a61fa6759e8612cc44d429af0417584807411e38b991b9bcedb9/vmlinuz-4.18.0-240.10.1.el8_3.x86_64 random.trust_cpu=on console=tty0 console=ttyS0,115200n8 ignition.platform.id=qemu ostree=/ostree/boot.0/rhcos/8db996458745a61fa6759e8612cc44d429af0417584807411e38b991b9bcedb9/0 root=UUID=a9fd5b50-93e0-4df8-975f-6e0d412d1183 rw rootflags=prjquota\""
time="2021-03-02T20:56:11+05:30" level=debug msg="(crc) DBG | time=\"2021-03-02T20:56:11+05:30\" level=debug msg=\"exe=/Users/redhat/.crc/bin/crc-driver-hyperkit uid=0\""
time="2021-03-02T20:56:11+05:30" level=debug msg="(crc) DBG | time=\"2021-03-02T20:56:11+05:30\" level=debug msg=\"hyperkit pid from json: 20684\""
time="2021-03-02T20:56:11+05:30" level=debug msg="(crc) DBG | time=\"2021-03-02T20:56:11+05:30\" level=debug msg=\"Searching for e:f0:e3:c7:4f:be in /var/db/dhcpd_leases ...\""
time="2021-03-02T20:56:11+05:30" level=debug msg="(crc) DBG | time=\"2021-03-02T20:56:11+05:30\" level=debug msg=\"Found 68 entries in /var/db/dhcpd_leases!\""
time="2021-03-02T20:56:11+05:30" level=debug msg="(crc) DBG | time=\"2021-03-02T20:56:11+05:30\" level=debug msg=\"dhcp entry: {Name:crc-l6qvn-master-0 IPAddress:192.168.64.49 HWAddress:e:f0:e3:c7:4f:be ID:1,e:f0:e3:c7:4f:be Lease:0x603fa6c4}\""
time="2021-03-02T20:56:11+05:30" level=debug msg="(crc) DBG | time=\"2021-03-02T20:56:11+05:30\" level=debug msg=\"Found match: e:f0:e3:c7:4f:be\""
time="2021-03-02T20:56:11+05:30" level=debug msg="(crc) DBG | time=\"2021-03-02T20:56:11+05:30\" level=debug msg=\"IP: 192.168.64.49\""
time="2021-03-02T20:56:11+05:30" level=debug msg="(crc) Calling .GetState"
time="2021-03-02T20:56:11+05:30" level=debug msg="(crc) DBG | time=\"2021-03-02T20:56:11+05:30\" level=debug msg=\"exe=/Users/redhat/.crc/bin/crc-driver-hyperkit uid=0\""
time="2021-03-02T20:56:11+05:30" level=debug msg="(crc) DBG | time=\"2021-03-02T20:56:11+05:30\" level=debug msg=\"hyperkit pid from json: 20684\""
time="2021-03-02T20:56:11+05:30" level=debug msg="(crc) Calling .GetIP"
time="2021-03-02T20:56:11+05:30" level=debug msg="Waiting until ssh is available"
time="2021-03-02T20:56:11+05:30" level=debug msg="retry loop: attempt 0"
time="2021-03-02T20:56:11+05:30" level=debug msg="Running SSH command: exit 0"
time="2021-03-02T20:56:11+05:30" level=debug msg="Using ssh private keys: [/Users/redhat/.crc/cache/crc_hyperkit_4.7.0/id_ecdsa_crc /Users/redhat/.crc/machines/crc/id_ecdsa]"
time="2021-03-02T20:57:12+05:30" level=debug msg="SSH command results: err: dial tcp 192.168.64.49:22: i/o timeout, output: "
time="2021-03-02T20:57:12+05:30" level=debug msg="error: Temporary error: ssh command error:\ncommand : exit 0\nerr     : dial tcp 192.168.64.49:22: i/o timeout\\n - sleeping 1s"
time="2021-03-02T20:57:13+05:30" level=debug msg="retry loop: attempt 1"
time="2021-03-02T20:57:13+05:30" level=debug msg="Running SSH command: exit 0"
time="2021-03-02T20:57:13+05:30" level=debug msg="Using ssh private keys: [/Users/redhat/.crc/cache/crc_hyperkit_4.7.0/id_ecdsa_crc /Users/redhat/.crc/machines/crc/id_ecdsa]"
time="2021-03-02T20:58:13+05:30" level=debug msg="SSH command results: err: dial tcp 192.168.64.49:22: i/o timeout, output: "
time="2021-03-02T20:58:13+05:30" level=debug msg="error: Temporary error: ssh command error:\ncommand : exit 0\nerr     : dial tcp 192.168.64.49:22: i/o timeout\\n - sleeping 1s"
time="2021-03-02T20:58:14+05:30" level=debug msg="retry loop: attempt 2"
time="2021-03-02T20:58:14+05:30" level=debug msg="Running SSH command: exit 0"
time="2021-03-02T20:58:14+05:30" level=debug msg="Using ssh private keys: [/Users/redhat/.crc/cache/crc_hyperkit_4.7.0/id_ecdsa_crc /Users/redhat/.crc/machines/crc/id_ecdsa]"
time="2021-03-02T20:59:14+05:30" level=debug msg="SSH command results: err: dial tcp 192.168.64.49:22: i/o timeout, output: "
time="2021-03-02T20:59:14+05:30" level=debug msg="error: Temporary error: ssh command error:\ncommand : exit 0\nerr     : dial tcp 192.168.64.49:22: i/o timeout\\n - sleeping 1s"
time="2021-03-02T20:59:15+05:30" level=debug msg="retry loop: attempt 3"
time="2021-03-02T20:59:15+05:30" level=debug msg="Running SSH command: exit 0"
time="2021-03-02T20:59:15+05:30" level=debug msg="Using ssh private keys: [/Users/redhat/.crc/cache/crc_hyperkit_4.7.0/id_ecdsa_crc /Users/redhat/.crc/machines/crc/id_ecdsa]"
time="2021-03-02T21:00:15+05:30" level=debug msg="SSH command results: err: dial tcp 192.168.64.49:22: i/o timeout, output: "
time="2021-03-02T21:00:15+05:30" level=debug msg="error: Temporary error: ssh command error:\ncommand : exit 0\nerr     : dial tcp 192.168.64.49:22: i/o timeout\\n - sleeping 1s"
time="2021-03-02T21:00:16+05:30" level=debug msg="retry loop: attempt 4"
time="2021-03-02T21:00:16+05:30" level=debug msg="Running SSH command: exit 0"
time="2021-03-02T21:00:16+05:30" level=debug msg="Using ssh private keys: [/Users/redhat/.crc/cache/crc_hyperkit_4.7.0/id_ecdsa_crc /Users/redhat/.crc/machines/crc/id_ecdsa]"
time="2021-03-02T21:01:16+05:30" level=debug msg="SSH command results: err: dial tcp 192.168.64.49:22: i/o timeout, output: "
time="2021-03-02T21:01:16+05:30" level=debug msg="error: Temporary error: ssh command error:\ncommand : exit 0\nerr     : dial tcp 192.168.64.49:22: i/o timeout\\n - sleeping 1s"
time="2021-03-02T21:01:17+05:30" level=debug msg="RetryAfter timeout after 5 tries"
time="2021-03-02T21:01:17+05:30" level=debug msg="Making call to close driver server"
time="2021-03-02T21:01:17+05:30" level=debug msg="(crc) Calling .Close"
time="2021-03-02T21:01:17+05:30" level=debug msg="Successfully made call to close driver server"
time="2021-03-02T21:01:17+05:30" level=debug msg="Making call to close connection to plugin binary"
time="2021-03-02T21:01:17+05:30" level=debug msg="(crc) DBG | time=\"2021-03-02T21:01:17+05:30\" level=debug msg=\"Closing plugin on server side\""
time="2021-03-02T21:01:17+05:30" level=error msg="Failed to connect to the CRC VM with SSH -- host might be unreachable: Temporary error: ssh command error:\ncommand : exit 0\nerr     : dial tcp 192.168.64.49:22: i/o timeout\\n (x5)"
```

**Comment by cfergeau** [comment]
The SSH connection fails because the guest os crashed with a kernel panic.
Duplicate of #2020
